src/google_apps_clj/google_sheets_v4.clj

```diff
@@ -238,3 +301,44 @@
                    (add-sheet-id))]
     (write-sheet service worksheet-id sheet-id table)
     sheet-id))
+
+(defn get-effective-vals
+  "sheet-ranges is a list of strings, using the A1 syntax, eg [\"Sheet!A1:Z9\"]
+  Returns a list of tables in corresponding to sheet-ranges.  Only one
+  sheet (tab) can be specified per batch, due to a quirk of Google's API as far
+  as we can tell."
+  [service worksheet-id sheet-ranges]
```

Comment:
I reckon I'd want a clojure form for sheet range expressions, e.g.:

`[sheet1 [0 0] [26 10]]`

Or maybe even a record/type, if there's meaningful variation in the constraints expressed by a sheet range expression...

Is there a sheet range expression for "gimme all the data on the sheet I don't care"?